### PR TITLE
Feature/#23 setting static model

### PR DESCRIPTION
**Pull request by harkenn**
Is one small piece of #23. Not insanely difficult, just a lot of changes. This is an equivalent exchange of the old settings and the new settings except for these major factors I have not (yet) addressed:

1. There is no migration to move tag data into `adventures.setting_id`
2. There is no migration to remove tag data rows where the field is 'setting' (after 1 is done)
3. Setting does not show up as a filter anymore

All three of these can be handled, I just didn't get to them tonight.

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -50,13 +50,10 @@ doctrine:
...
-        # if using pdo_sqlite as your database driver:
-        #   1. add the path in parameters.yml
-        #     e.g. database_path: "%kernel.root_dir%/../var/data/data.sqlite"
-        #   2. Uncomment database_path in parameters.yml.dist
-        #   3. Uncomment next line:
-        #path: '%database_path%'
+        charset: utf8mb4
+        default_table_options:
+            charset: utf8mb4
+            collate: utf8mb4_unicode_ci
```
Comment (cmfcmf) [inline app/config/config.yml]: 👍

File: src/AppBundle/Service/AdventureSerializer.php [hunk 2]
```diff
@@ -1,10 +1,4 @@
 <?php
-/**
- * Created by PhpStorm.
- * User: cmfcm
- * Date: 13.04.2017
- * Time: 19:06
- */
```
Comment (cmfcmf) [inline src/AppBundle/Service/AdventureSerializer.php]: 👍

**Comment by cmfcmf** [comment]
I didn't want to push to your branch, so I pushed a commit in my repository showing how a basic setting selector can be added to a Symfony form (in this case, the edit adventure form): https://github.com/cmfcmf/AdventureLookup/commit/4c9c42a1e888015ff9a9411d8b995d7f89787efe

**Comment by harkenn** [comment]
Thanks I'll implement this today so

**Comment by harkenn** [comment]
@cmfcmf Its been updated. Be sure when you review, if you accept the review you have to hit "approve" so that it can be merged.

**Comment by harkenn** [comment]
I'm torn on writing migrations for one-off stuff that technically is only relevant in the production environment. If we were standing up a new production system, we would look back on this migration and be like "hrm, that's irrelevant." This is driven mostly from mixed experience with this approach.

**Comment by harkenn** [comment]
Ok did a quick test, having the code changes out but the schema/tables still on utf8 doesn't seem to do any harm. So we could technically roll out the utf8mb4 changes all the way out to production and not have any issues. So we'll definitely do the conversions on the production server manually.

**Pull request merged**